tests/roots/test-latex-figure-in-admonition/conf.py

```diff
@@ -0,0 +1,3 @@
+master_doc = 'index'
```

Comment:
By default, `master_doc` equals `'index'`. So this is not needed here.

Reply:
Of course, removed